price-feeder/oracle/provider/coinbase.go

```diff
@@ -94,11 +96,19 @@ type (
 func NewCoinbaseProvider(
 	ctx context.Context,
 	logger zerolog.Logger,
+	endpoints config.ProviderEndpoint,
 	pairs ...types.CurrencyPair,
 ) (*CoinbaseProvider, error) {
+	if endpoints.Name != config.ProviderCoinbase {
+		endpoints = config.ProviderEndpoint{
+			Name:      config.ProviderCoinbase,
+			Rest:      coinbaseRestHost,
+			Websocket: coinbaseWSHost,
+		}
+	}
```

Comment:
This condition could throw an error instead if we wanted to avoid silently defaulting when passing the wrong (as opposed to just empty) endpoint.